docs-release-notes.md

```diff
@@ -0,0 +1,60 @@
+# June 10, 2022
+
+The App Services docs release notes start here from week ending May 27, 2022. Prior to this, the App Services docs release notes were presented alongside the Realm docs release notes in the [Realm Docs repository](https://github.com/mongodb/docs-realm/blob/master/docs-release-notes.md).
```

Comment:
```suggestion
The App Services docs release notes start here from week ending May 27, 2022. Prior to this, the App Services docs release notes were presented alongside the Realm docs release notes in the [Realm SDK Docs repository](https://github.com/mongodb/docs-realm/blob/master/docs-release-notes.md).
```
While the repo is literally called "docs-realm", it might be helpful to clarify that that repo is now used for SDK docs exclusively.